alibi_detect/cd/keops/learned_kernel.py

```diff
@@ -101,7 +101,7 @@ def __init__(
         epochs
             Number of training epochs for the kernel. Corresponds to the smaller of the reference and test sets.
         num_workers
-            Number of workers for the dataloader.
+            Number of workers for the dataloader. Note: Setting `num_workers>0` may be unreliable on Windows.
```

Comment:
>1 instead of >0?

Reply:
I'm saying `>0` because the default `0` means multi-processing data loading is turned off (from [torch docs](https://pytorch.org/docs/stable/data.html#single-and-multi-process-data-loading)). `>0` causes errors on Windows. We could change the description to `>=1` if that is clearer?

Reply:
Ok got it, that's fine then. Was confused because initially in the PR it said issues were caused on windows > 1.

Reply:
Yeh sorry noticed my typo after you commented 🙈

Reply:
Updated default to `num_workers=0` in ffe1b78